packages/web-twig/src/Resources/components/Accordion/AccordionContent.twig

```diff
@@ -0,0 +1,37 @@
+{# API #}
+{%- set props = props | default([]) -%}
+{%- set _class = props.class | default(null) -%}
+{%- set _id = props.id -%}
+{%- set _labelledById = props.labelledById -%}
+{%- set _parent = props.parent | default(null) -%}
+{%- set _isOpen = props.isOpen | default(false) | boolprop -%}
+
+{# Class names #}
+{%- set _rootClassName = _spiritClassPrefix ~ 'Collapse' -%}
+{%- set _collapseClassName = _spiritClassPrefix ~ 'Collapse__content' -%}
+{%- set _accordionClassName = _spiritClassPrefix ~ 'Accordion__content' -%}
+{%- set _isOpenClassName = _isOpen ? 'is-open' : null -%}
+
+{# Attributes #}
+{%- set _idAttr = _id ? 'id="' ~ _id ~ '"' : null -%}
+{%- set _labelledIdAttr = _labelledById ? 'aria-labelledby="' ~ _labelledById ~ '"' : null -%}
```

Comment:
I believe we always try (though maybe imperfectly) to mirror the attribute's name into the variable name here:

```md
              ↓ <targetAttributeName>Attr   ↓ <condition>   ↓ <target-attribute>   ↓ <_privatePropName>
```

```suggestion
{%- set _ariaLabelledByAttr = _labelledById ? 'aria-labelledby="' ~ _labelledById ~ '"' : null -%}
```

It's not that important, but still, it's one of those small things that help us keep the code consistent 🙂.

Reply:
ok, fixed